python/taichi/lang/ast/ast_transformer_utils.py

```diff
@@ -177,7 +178,10 @@ def get_var_by_name(self, name):
         for s in reversed(self.local_scopes):
             if name in s:
                 return s[name]
-        return self.global_vars.get(name)
+        var = self.global_vars.get(name)
+        if var:
+            return var
+        return getattr(builtins, name, None)
```

Comment:
```python
return var or getattr(builtins, name, None) # maybe better?
```

Reply:
This might be buggy if var equals to 0?

Reply:
yes I did not see your second commit(^-^)